kasa/discover.py

```diff
@@ -133,6 +137,46 @@
 }
 
 
+class _AesDiscoveryQuery:
+    keypair = None
```

Comment:
Maybe store this also inside experimental and/or in a separate file.

Reply:
I think we should keep this so discovering an unsupported device reports nicely. i.e. this specifically is not experimental.

Reply:
Agreed, if we want to clean the discovery, we can move all of it into separate files under the `discovery` package or something. Perhaps something worth considering also for transports (`transport/foo.py`).